manifests/m/ManicTime/ManicTime/24.3.3.0/ManicTime.ManicTime.installer.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ManicTime.ManicTime
+PackageVersion: 24.3.3.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: user

```

Comment:
This is because the installer does not specify an `ALLUSERS` value and the [documentation](https://learn.microsoft.com/en-gb/windows/win32/msi/allusers#default-value) for this specifies that if no value is set, it installs in a per-user context. This is a little confusing though because the install directory is %ProgramFiles%

Reply:
> Is there a way that it can look back at the previous manifest? Or would this cause further issues?

komac can, and does, but oftentimes previous manifests are incorrect and komac is able to get more accurate information from the binary itself and therefore fix the incorrect field. It would be better for me to figure out a way to deal with this specifically. For example, it could check the install directory for the scope as a backup? If it's installed to ProgramFiles, it's unlikely to actually be a user install scope.